examples/acados_python/getting_started/minimal_example_closed_loop.py

```diff
@@ -55,12 +55,14 @@
 N = 20
 
 # set dimensions
-ocp.dims.nx  = nx
-ocp.dims.ny  = ny
-ocp.dims.ny_e = ny_e
-ocp.dims.nbu = nu
-ocp.dims.nu  = nu
-ocp.dims.N   = N
+ocp.dims.N = N
+# NOTE: all dimensions but N are now detected automatically in the
```

Comment:
What about soft constraints, are they detected? Today I realized soft constraints are not detected also in the matlab interface ATM.

Reply:
In Python, I also did it for the slack dimensions, here:
https://github.com/FreyJo/acados/blob/b60dbfa922e48e4ab7bd849f2cca0a7b57f26076/interfaces/acados_template/acados_template/acados_ocp_solver.py#L224-L367

I think it would be great to also do this in Matlab.